### ISSUE TITLE
Resources beyond single talk topics

### ISSUE THREAD

**Issue by eddelbuettel** [ess-intro/ess-intro#5]
Hm, I am not convinced yet that https://github.com/ess-intro/ess-intro/wiki/Resources is a winner.  I fear this could turn a random drop (and "superset" of topic talks).

@aezarebski  Thoughts?

We do want supporting documentation to go along with the talks.  One thing I just realized is that ess-intro could (and probably should) have a `docs/` folder served via gh-pages.

**Comment by aezarebski** [comment]
A random dumping ground was exactly what I intended it to be. Rather than clogging up the topics discussion page, I wanted a place to put _that link that seems super dooper important but is in reality just noise_. I'm not wedded to the idea though, so feel free to remove it.

**Comment by eddelbuettel** [comment]
Agreed that we probably want it. How about if I space it off from the 'On-going discussions' paragraph it is squatting in, and we add a new para 'Misc support' ?  (And make it a one-word link 'Resources')

**Comment by eddelbuettel** [comment]
And one point I also realized we may even need things like a glossary (_i.e._ not everybody who comes here may yet know what 'org' is (in the org-mode context)).

**Comment by aezarebski** [comment]
A glossary is an excellent idea, I still need to keep reminding myself of the "windows"/"frames" "buffer"/"file" distinctions.

**Comment by eddelbuettel** [comment]
Wiki updated. Lemme retitle this for keeping it open.
